app/assets/javascripts/discourse/app/mixins/composer-video-thumbnail-uppy.js

```diff
@@ -37,6 +37,7 @@ export default class ComposerVideoThumbnailUppy extends EmberObject.extend(
     super(...arguments);
     this.capabilities = owner.lookup("service:capabilities");
```

Comment:
do you know why caps isn't a service injection here?

Reply:
No idea, but will keep as-is for now. I'm trying to keep the diff to a minimum here for easy review. Might do a pass of improvements in a followup PR.